openlcs/packages/views.py

```diff
@@ -912,6 +912,11 @@ def post(self, request, *args, **kwargs):
 
         if serializer_data and len(serializer_data) == 1:
             obj = serializer_data[0]
+            source_ready = 'source' in obj and \
+                obj['source'] is None and parent != ''
```

Comment:
I tested importing the same rpm, and it works.  If we need improve the PR, late will file a new issue to update it.

Reply:
@hanchuntao As to what you commented, `parent != ''` is to indicate a task as a forked task under a container/module import. Also, if for a component, we still dont need to give the parent: if the `source_ready` is False due to `parent == ''`, it will skip the `source_ready` check but the following check will consider it as a duplicate task. I tested for component / container / module, it seems to work well.